library/cmac.c

```diff
@@ -909,7 +891,7 @@ int mbedtls_cmac_self_test( int verbose )
     /* AES-192 */
```

Comment:
A compilation warning is generated in line 879 because a cast is needed for `aes_128_key` to `const unsigned char *`.